gcp/functions/pypi/main.py

```diff
@@ -52,6 +52,15 @@ def publish(event, context):
       'versions': list(vulnerability.affects.versions),
       'link': f'https://osv.dev/vulnerability/{vulnerability.id}',
       'aliases': list(vulnerability.aliases),
+      'details': vulnerability.details,
+      'ranges': [
+          {
+              'introduced': r.introduced,
+              'fixed': r.fixed,
+          }
+          for r in vulnerability.affects.ranges
```

Comment:
`affects` is deprecated and will be removed soon. Could you get this from the `affected` field instead (https://ossf.github.io/osv-schema/#affected-fields)? (We also need to update the `'project'`  field above to use that as well because `package` was also moved into `affected`).